ONSdigital/dp-frontend-renderer
# Feature/simple filter

### What

Hard coded html structure for the filter list. Includes meta data element. 

There is a related pull request for the changes made to the sixteens repo - https://github.com/ONSdigital/sixteens/pull/32

### How to review

- To view you need to pull the corresponding "feature/simple-filter" branch from sixteens repo.
- View the template in browser via - http://localhost:20001/datasets/1/editions/2/versions/3
- Click the start button to view the filter list.

- Click the meta data element and it reveals hidden content.
- Click the select/deselect elements. Checkboxes should check/uncheck.
- Click checkboxes individually.
- Check against design/[alpha](https://discovery.onsdigital.co.uk/dd/datasets/ProdCom/editions/2016/versions/0/dimensions/Prodcom%20Elements).
- Test mobile for layout.
- Cross browser test for layout functionality.
- Accessibility test.


### Who can review

Front end, UX
